pkg/controllers/management/auth/globalroles/globalrole_handler.go

```diff
@@ -362,7 +362,7 @@ func (gr *globalRoleLifecycle) purgeInvalidNamespacedRoles(roles []*v1.Role, uid
 func (gr *globalRoleLifecycle) setGRAsInProgress(globalRole *v3.GlobalRole) error {
 	globalRole.Status.Conditions = []metav1.Condition{}
 	globalRole.Status.Summary = SummaryInProgress
-	globalRole.Status.LastUpdate = time.Now().String()
```

Comment:
We probably just could've used `Format()` without changing the field type

```go
globalRole.Status.LastUpdate = time.Now().Format(time.RFC3339)
```
and even normalize it by always using UTC timezone if we wanted to

```go
globalRole.Status.LastUpdate = time.Now().UTC().Format(time.RFC3339)
```